ui/tests/acceptance/auth/enable-tune-form-test.js

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { v4 as uuidv4 } from 'uuid';
+
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
+import { visit } from '@ember/test-helpers';
+import { deleteAuthCmd, runCmd } from 'vault/tests/helpers/commands';
+import testHelper from './test-helper';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+
+// These models use openAPI so we assert the form inputs using an acceptance test
+// The default selector is to use GENERAL.inputByAttr()
+// custom fields should be added to the this.customSelectorss object
+module('Acceptance | auth enable tune form test', function (hooks) {
```

Comment:
I only added tests for azure, jwt, oidc, okta and ldap for now. We can expand as needed